problem/problem_info.py

```diff
@@ -0,0 +1,29 @@
+import os.path
+from utils import config_info
+from problem.models import Problem, Testcase
+from django.db.models import Q
```

Comment:
Fine, this is a weird naming in my opinion XD
But it is ok if the package creates this kind of naming